langchain/vectorstores/milvus.py

```diff
@@ -803,3 +810,14 @@ def from_texts(
         )
         vector_db.add_texts(texts=texts, metadatas=metadatas)
         return vector_db
+
+    def _similarity_search_with_relevance_scores(
+        self,
+        query: str,
+        k: int = 4,
+        **kwargs: Any,
+    ) -> List[Tuple[Document, float]]:
+        """Return docs and their similarity scores on a scale from 0 to 1."""
+
+        docs_and_scores = self.similarity_search_with_score(query, k=k)
```

Comment:
do we know what scoring function is used by default?

Reply:
as in what is the unit of original score returned by milvus

Reply:
i mean what's the scoring metric. is it L2 distance, cosine similarity, inner product, etc. from docs seems like it's one of L2 and IP https://milvus.io/docs/metric.md#floating. wonder if there's some way to normalize that's more metric-aware

Reply:
<img width="719" alt="image" src="https://github.com/hwchase17/langchain/assets/49740903/4dc4eb1d-e41c-42c7-9c6d-9b0f9f476ca3">
<img width="756" alt="截屏2023-05-19 10 00 51" src="https://github.com/hwchase17/langchain/assets/49740903/c153c0db-ee5b-411e-8d1b-abb453ce1a1e">
I added the parameters of the normalized score calculation method to the constructor of the Milvus class. When calling, it will be judged whether the variable is empty. Like the above, I don’t know if this meets your needs.